src/main/java/nl/hsac/fitnesse/fixture/slim/web/SeleniumDriverSetup.java

```diff
@@ -198,6 +220,32 @@ public boolean connectToDriverAtWithCapabilities(String url, Map<String, Object>
         return createAndSetRemoteDriver(url, desiredCapabilities);
     }
 
+    public boolean connectToFireFoxDriverAtWithProfile(String url, Map<String, String> profile)
```

Comment:
Shouldn't this be called connectToFirefoxDriverAtWithProfile (no capital F in fox)